modules.json

```diff
@@ -10,12 +10,11 @@
     "concertBidAdapter",
     "connectIdSystem",
     "consentManagementUsp",
-    "consentManagement",
+    "consentManagementTcf",
     "conversantBidAdapter",
     "criteoBidAdapter",
     "criteoIdSystem",
     "dfpAdServerVideo",
-    "enrichmentFpdModule",
```

Comment:
Has been removed in [Prebid 9.0](https://github.com/prebid/Prebid.js/pull/11720)